bx_django_utils/models/manipulate.py

```diff
@@ -23,14 +23,14 @@ class InvalidStoreBehavior(FieldDoesNotExist):
     pass
 
 
-def create(*, ModelClass, call_full_clean=True, **values):
+def create(*, ModelClass, call_full_clean=True, save_kwargs={}, **values):
```

Comment:
Bad design to set a dict as default. Please use None as default.

Reply:
Why is this a problem here? I'm aware of problems when mutating the kwargs, but they are just passed, thereby copying them. But whatever, I can change it.